virttest/vt_resmgr/resources/storage/file_volume.py

```
@@ -0,0 +1,231 @@
+            raise ValueError(f"New volume size is not set")
+
+        bsize = int(normalize_data_size(size, "B"))
+        if bsize != self.spec["size"]:
+            node, backing_id = self.get_binding(node)
+            r, o = node.proxy.resource.update_resource_by_backing(
+                backing_id,
+                "resize",
+                arguments,
+            )
+            if r != 0:
+                raise Exception(o["out"])
```

Ruff (0.13.1)

Create your own exception

(TRY002)